core/src/test/java/org/mule/tck/SensingNullRequestResponseMessageProcessor.java

```diff
@@ -35,4 +38,21 @@ protected MuleEvent processResponse(MuleEvent event) throws MuleException
         return super.processRequest(event);
     }
 
+    public static void assertSameThread(MuleContext muleContext, String beanName) throws Exception
```

Comment:
These methods don't belong here.  It's a good idea to have an assertResponeRequestThreadsDifferent/assertResponeRequestThreadsSame here, but it' should take no parameters and rather do lookup etc.